provision/ovh.go

```diff
@@ -75,8 +76,32 @@ func (o *OVHProvisioner) Status(id string) (*ProvisionedHost, error) {
 	}, nil
 }
 
+func (o *OVHProvisioner) lookupID(request HostDeleteRequest) (string, error) {
+	inlets, err := o.client.ListInstance(context.Background())
```

Comment:
Can you change `inlets` and `inlet` to `instances` and `instance` please?

Reply:
The sdk returns as "error" 

`return "", errors.Errorf("no ip4 address found for instance: %s", instance.Name)`

So it would be a "duplication" of the messages, as `lookupID` has his on error at the end.